### ISSUE TITLE
Implement `QueryablePayload::transaction_with_proof` for multiple namespaces

### ISSUE THREAD

**Issue by ggutoski** [EspressoSystems/espresso-sequencer#1010]
Next step after #991 , #931 .

- [ ] Add comment related to empty name spaces and basic tests. See https://github.com/EspressoSystems/espresso-sequencer/issues/1043#issuecomment-1913768156

**Comment by philippecamacho** [comment]
Low priority as it does not block other tasks. Needed only as a feature for end users.

**Comment by philippecamacho** [comment]
@jbearer do we need this feature for Cappuccino sprint 2?
